code-challenges.test.js

```diff
@@ -22,25 +36,181 @@ const people = [
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 // b) Create the function that makes the test pass.
+// psuedo code - i am taking an array and capping the first leatter of every index/ this is an array , of ill break it down and use Object.values to manupulate the text. so i used a lot per array index. i usedtoString to tur the array into a astring then used the replace function and replace the lowered cased names nd turn the first latter of them to Capital.after the values were replaced i split it back into a array index and the called all of the array index togother by con cating them into newArray and returnin it.
```

Comment:
You have a couple of ideas that can work for this function you are trying to create! You are correct in that you need to capitalize the first letter but not for every index in the initial array we are passing. Keep in mind that we have an array with Objects and the name is what we want to capitalize. I see that you mentioned split and Object.values and you are on the right track in that we need to target a value within the object and maybe, we can split up name and capitalize the first index in each name, first and last. I love the fact you are using console logs to see what you are working with at each step, I would definitely give this one another try! 👏🏽